docker/pypi/wmagent/install.sh

```diff
@@ -176,7 +189,7 @@ echo "Start $stepMsg"
 chmod +x $WMA_DEPLOY_DIR/deploy/renew_proxy.sh $WMA_DEPLOY_DIR/deploy/restartComponent.sh
 
 crontab -u $WMA_USER - <<EOF
-55 */12 * * * $WMA_DEPLOY_DIR/deploy/renew_proxy.sh
+55 */12 * * * $WMA_MANAGE_DIR/manage renew-proxy
```

Comment:
Oh, this is where the continuous proxy renewal process is defined.

Reply:
yep :)

Reply:
And the reason why it lives in `install.sh` instead of `run.sh`, is because it is a predefined logic independent of any runtime parameter. So no need to be part of the `init` process at all. It is is easily definable at build time. I may suggest while you are looking to the rest of the PR, if you notice there is more of the logic, which is independent of any runtime parameters or initial configuration parameters, to point them and we can make the effort to move them here as well.